src/tools/interop/interop.cpp

```diff
@@ -784,19 +816,28 @@ RunInteropTest(
                 Success = Connection.SendHttpRequests();
             }
         }
+        if (ResumptionTicket) {
+            delete [] ResumptionTicket;
+        }
         break;
     }
 
     case StreamData:
     case ZeroRtt: {
+        const uint8_t* ResumptionTicket = nullptr;
+        uint32_t ResumptionTicketLength = 0;
         if (Feature == ZeroRtt) {
             InteropConnection Connection(Configuration);
             if (!Connection.ConnectToServer(Endpoint.ServerName, Port) ||
-                !Connection.WaitForTicket()) {
+                !Connection.WaitForTicket() ||
+                !Connection.GetResumptionTicket(ResumptionTicket, ResumptionTicketLength)) {
                 break;
             }
         }
```

Comment:
Might be worth just making a helper function for this to reduce duplication.

Reply:
If we have a third test case that needs to do this, I'll agree. But for now, this isn't so bad.